### PR TITLE
Map component tests

### PR DESCRIPTION
**Pull request by avanti-em**

### CODE REVIEW

File: react/src/lib/components/DeckGLMap/components/settings/DrawModeSelector.test.tsx [hunk 1]
```diff
@@ -6,9 +6,10 @@ import { testStore, Wrapper } from "../../test/TestWrapper";
 import DrawModeSelector from "./DrawModeSelector";
 
 describe("Test draw-mode menu", () => {
+    // The render function is rendering just 'null'
     it("snapshot test", () => {
         const { container } = render(
-            Wrapper({ children: <DrawModeSelector layerId="colormap-layer" /> })
+            Wrapper({ children: <DrawModeSelector layerId="" /> })
```
Comment (alexandruandrei91) [inline react/src/lib/components/DeckGLMap/components/settings/DrawModeSelector.test.tsx:12]: The `layerId` prop needs to match the DrawingLayer `id` field from deckgl-map.json. Now it is `"drawing-layer"`.
Same for the rest of DrawModeSelector components in this file.

File: react/src/lib/components/DeckGLMap/components/settings/LayerSettingsButton.test.tsx [hunk 2]
```diff
@@ -6,14 +6,12 @@ import { testStore, Wrapper } from "../../test/TestWrapper";
...
+    // render function is rendering just 'null'
     it("snapshot test", () => {
         const { container } = render(
             Wrapper({
                 children: (
-                    <LayerSettingsButton
-                        layerId="colormap-layer"
-                        layerType="ColormapLayer"
-                    />
+                    <LayerSettingsButton layerId="" layerType="ColormapLayer" />
```
Comment (alexandruandrei91) [inline react/src/lib/components/DeckGLMap/components/settings/LayerSettingsButton.test.tsx:14]: The `layerId` prop needs to match the ColormapLayer `id` field from deckgl-map.json. Now it is `"colormap-layer"`.
Same for the rest of LayerSettingsButton components in this file.

**Review by alexandruandrei91 — COMMENTED** [review]
You might also get a problem with the LayersButton. In that case, you need to add this to LayersButton.tsx:
```js
import { layers } from "@equinor/eds-icons";

Icon.add({ layers });
```

Let me know if you bump into other problems :)

File: react/src/lib/components/DeckGLMap/components/settings/DrawModeSelector.test.tsx (hunk 1 above)
Comment (avanti-em) [reply react/src/lib/components/DeckGLMap/components/settings/DrawModeSelector.test.tsx]: Thank you so much @alexandruandrei91 for solving all these issues :)
The snapshots are all fine now.

**Review by alexandruandrei91 — COMMENTED** [review]
👍

**Review by alexandruandrei91 — APPROVED** [review]

**Pull request merged**